server/test/src/scala/lsug/ServerSpec.scala

```diff
@@ -0,0 +1,86 @@
+package scala.lsug
+
+import cats.effect.{Blocker, IO}
+import lsug.protocol.Meetup.MeetupDotCom
+import lsug.protocol.Meetup.MeetupDotCom.Event
+import lsug.{IOSuite, Meetup, Server, protocol}
+
+import java.nio.file.Paths
+import java.time.{LocalDate, LocalDateTime, LocalTime, ZoneId, ZonedDateTime}
+
+final class ServerSpec extends IOSuite {
+
+  class MockedMeetup extends Meetup[IO] {
+    override def event(id: Event.Id): IO[Option[MeetupDotCom.Event]] = IO(None)
+  }
+  val mockedMeetup = new MockedMeetup
+
+  test("eventsBefore return a list of past events ordered by time") {
+
+    Blocker[IO].use { blocker =>
+      val server: Server[IO] = new Server[IO](
+        Paths.get("./server/src/main/resources").toAbsolutePath,
+        mockedMeetup,
+        blocker
+      )
+      val result = server.eventsBefore(
+        ZonedDateTime.of(
+          LocalDate.now(),
+          LocalTime.now(),
+          ZoneId.of("UTC")
+        )
+      )
+
+      result.map { events =>
+        val listOfEventTimes: List[LocalDateTime] =
+          events.map(e => e.setting.time.start)
+
+        def checkSorting(eventTimes: List[LocalDateTime]): Unit = {
+          eventTimes.foldLeft(LocalDateTime.now())((ctx, x) =>
```

Comment:
Munit uses the variable names as clue names, so it's worth thinking a bit about the names here.  What do you think about `timeAbove` and `timeBelow`, to signal their visual order in the list?

You can also use an or `||` instead of an `if`:

```scala
eventTimes.foldLeft(localTimeLondon)(
  (timeAbove, timeBelow) => {
    assert(
      clue(timeAbove).isEqual(clue(timeBelow)) || timeAbove
        .isAfter(timeBelow)
    )
    timeBelow
  }
)
```